tests/active-active-rhel7-proxy/main.tf

```diff
@@ -96,7 +96,7 @@ module "tfe" {
   node_count                  = 2
   proxy_ip                    = module.test_proxy.proxy_ip
   proxy_port                  = local.http_proxy_port
-  redis_encryption_at_rest    = false
+  redis_encryption_at_rest    = true
```

Comment:
💭 We could also set the default value to true in addition to explicitly setting it. That might be a nice safeguard mechanism.

Reply:
Done... unsure if that'll upset anyone externally using this.